webpack.config.js

```diff
@@ -164,7 +164,7 @@ const config = (module.exports = {
       __support__: TEST_SUPPORT_PATH,
       e2e: E2E_PATH,
       style: SRC_PATH + "/css/core/index",
-      ace: __dirname + "/node_modules/ace-builds/src-min-noconflict",
```

Comment:
> 2) As we can't yet transform completely to src-min-noconflict completely

Why not?

`src-min-noconflict` has been used [since forever](https://github.com/metabase/metabase/commit/7af54fa27d270230e0268e0a02fc2eb26868d381).
I've been trying to understand why we're using `src-noconflict` and got as far as this: #33102
I don't see a reason why we used `src-noconflict` there.

Reply:
I don't know the answer to this question either. @kulyk can you explain if `src-noconflict` is needed over `src-min-noconflict`?

Reply:
TBH I think I just copied it from [`ExpressionEditorTextfield`](https://github.com/metabase/metabase/blob/a2d3b0052093e5def5a5d66d9d9b2fca55aef05a/frontend/src/metabase/query_builder/components/expressions/ExpressionEditorTextfield/ExpressionEditorTextfield.tsx#L7) to make TypeScript happy, there's no intention in picking `src-noconflict` here

Reply:
Is there any reason to prefer `src-min-noconflict` over `src-noconflict`? It all gets minified in the build anyway

Reply:
Not sure, but if both are eventually minified, `src-noconflict` could be more handy in development if we need to look something up in ace's source code

Reply:
> It all gets minified in the build anyway

This could affect build speed though - worth measuring.

Reply:
> leave it for another PR

Done [src-min-noconflict -> src-noconflict](https://github.com/metabase/metabase/pull/34602#top)

Reply:
If you see what webpack does on yarn dev, we end up pulling src-min-noconflict and also with src-noconflict, 2 deps that are exactly the same, one minified, one not. By using always src-noconflict we ensure we pull the dep just once as webpack will end up minifying it on production builds. My change was only to ensure we pull things just once and reuse it as much as possible

Reply:
> As we can't yet transform completely to src-min-noconflict completely

@paoliniluis Can you please explain why we can't transform completely to src-min-noconflict?

Reply:
I have my suspicions that the aliasing in the webpack config doesn’t work, at least when you do yarn , you still see both deps being pulled and separate